### PR TITLE
fix: santander is not a valid IBAN

### PR DESCRIPTION
**Pull request by lippertto**

### CODE REVIEW

**Comment by lippertto** [comment]
Hello @kewisch , this PR adds jest for unittesting and fixes #18

File: .eslintrc [hunk 1]
```diff
@@ -1,4 +1,6 @@
 {
+    "extends": ["eslint:recommended"],
+    "parser": "@babel/eslint-parser",
```
Comment (kewisch) [inline .eslintrc:3]: Do we need the custom parser here or can we go with the default? If it is a missing js feature, you might get away with just changing ecmaVersion to catch up.

Comment (lippertto) [reply .eslintrc:3]: I just checked. If I set `es6` in `env` to `true`, then I do not need the parser from babel.
Is `es6` fine for you? If yes, I'll provide a patch to remove the babel parser and set the corresponding eslint option.

Comment (kewisch) [reply .eslintrc:3]: You should be able to get by by adding `"es2024": true` to the `env` section, no parser at all.

Comment (lippertto) [reply .eslintrc:3]: Yes, exactly. Is there any specific ES version that you would like to target in the project? If not, I will just use the lowest one that allows `eslint` to work, i.e., `es6: true`

Comment (kewisch) [reply .eslintrc:3]: Oh duh, sorry, I misread your comment, for some reason I was thinking you'd be using a parser named es6. Go ahead with es6: true in env!

Comment (lippertto) [reply .eslintrc]: Okay, I have added the changes, i.e., added `es6: true` and removed all traces of `@babel/eslint-parser`

**Comment by lippertto** [comment]
Is everything ok with this PR? I would like to have a look at #58 , and it would be good to have jast available for that.

**Comment by kewisch** [comment]
Looks great, thanks!

**Pull request merged**